### ISSUE TITLE
[BUG] `bg-center` mark as conflicting with `bg-[image:xxx]`

### ISSUE THREAD

**Issue by moshest** [francoismassart/eslint-plugin-tailwindcss#260]
**Describe the bug**
I want to setup both background image and positioning but I get error of `tailwindcss/no-contradicting-classname`.

I need to use CSS vartiables as I want the URL to be dynamic ([see here](https://stackoverflow.com/a/70810692/518153)).

**To Reproduce**

Use the following code:

```tsx
<section
    style={{ '--bg-small': `url(foo.jpg)` } as CSSProperties}
    className={`bg-[image:var(--bg-small)] bg-center`}
  />
```

**Expected behavior**
Code should work without an error.

**Additional context**
`bg-[url(...)]` with `bg-center` works fine though.
